FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/materials.xml

```diff
@@ -475,4 +475,35 @@
     <fraction n="0.084" ref="Nb"/>
   </material>
 
+
+  <material name="Water">
+      <D value="1" unit="g/cm3" />
+      <composite n="2" ref="H" />
+      <composite n="1" ref="O" />
+  </material>
+
+  <material name="DefaultMaterial">
+    <D value="7.85" unit="g/cm3"/>
+    <fraction n="0.998" ref="Fe"/>
+    <fraction n=".002"  ref="C"/>
+  </material>
+  
+  <material name="Material01">
```

Comment:
Maybe call it `IronDens785` or something?

Reply:
I think I inherited that material from some version of the `materials.xml` file. In this folder it is not called from anything so I remove it (same for `IDEA_o1_v03/materials_o1_v02.xml`)

Reply:
changed name in both files